Modules/Filtering/ImageIntensity/test/itkMaskNegatedImageFilterTest.cxx

```diff
@@ -156,6 +159,42 @@ itkMaskNegatedImageFilterTest(int, char *[])
     }
   }
 
+  // Vector image tests
+  using myVectorImageType = itk::VectorImage<float, myDimension>;
+
+  auto inputVectorImage = myVectorImageType::New();
+  inputVectorImage->SetRegions(region);
+  inputVectorImage->SetNumberOfComponentsPerPixel(3);
+  inputVectorImage->Allocate();
+
+  using myVectorFilterType = itk::MaskNegatedImageFilter<myVectorImageType, MaskImageType, myVectorImageType>;
+
+  auto vectorFilter = myVectorFilterType::New();
+  vectorFilter->SetInput1(inputVectorImage);
+  vectorFilter->SetMaskImage(inputMask);
+
+  myVectorImageType::PixelType outsideValue = vectorFilter->GetOutsideValue();
+  ITK_TEST_EXPECT_EQUAL(outsideValue.GetSize(), 0);
+
+
+  ITK_TRY_EXPECT_NO_EXCEPTION(vectorFilter->Update());
+
+  // Check that the outside value consists of three zeros.
+  myVectorImageType::PixelType outsideValue3 = vectorFilter->GetOutsideValue();
+  myVectorImageType::PixelType threeZeros(3);
+  threeZeros.Fill(0.0f);
+  ITK_TEST_EXPECT_EQUAL(outsideValue3, threeZeros);
+
+  // Reset the outside value to zero vector of length 23.
+  myVectorImageType::PixelType zeros23(23);
+  zeros23.Fill(1.0f);
+  vectorFilter->SetOutsideValue(zeros23);
+  std::cerr << zeros23 << std::endl;
```

Comment:
Why print to `cerr`? This is expected, so it should go to `cout`?

Reply:
Those line were used for debugging and can be removed.

I used std:err because as I recall it interacts with ctest output a little better.